{{cookiecutter.app_name}}/docker-compose.yml

```
@@ -45,7 +45,7 @@ services:
       target: development
       args:
         <<: *build_args
-    entrypoint: flask
+    entrypoint: pipenv run flask
```

Thanks for the fix! Using `pipenv` is optional, so we'd want to add templating like the following such that the additional command is only applied for users who opted in to pipenv.

```suggestion
    entrypoint: {% if cookiecutter.use_pipenv == 'True' %}pipenv run {% endif %}flask
```